src/aleph/vm/controllers/firecracker/instance.py

```diff
@@ -79,6 +79,7 @@ def __init__(
             enable_console,
             hardware_resources or MachineResources(),
             tap_interface,
+            True,
```

Comment:
The last argument is not trivial.

```suggestion
            persistent=True,
```